Lib/sqlite3/dbapi2.py

```diff
@@ -84,6 +84,20 @@ def convert_timestamp(val):
 
 register_adapters_and_converters()
 
+# bpo-24464: enable_shared_cache was deprecated in Python 3.10.  It's
+# scheduled for removal in Python 3.12.
+_old_enable_shared_cache = enable_shared_cache
+def enable_shared_cache(enable):
```

Comment:
I think this idiom is not reload-safe (I remember changes from `_open = open` to using `builtins.open`).

You can try this:

```python
from _sqlite3 import enable_shared_cache as _old_enable_shared_cache
```

Reply:
Of course, I should've thought of that. Thanks!